README.md

```diff
@@ -3,19 +3,35 @@ The new and improved DDW Analyst UI interface
 
 ## Setup
 
-<!-- 1. Make your database `sudo su postgres -c 'psql -U postgres -d postgres -c '"'"'CREATE DATABASE ddw_metadata'"'"`
-2. Make sure your psql database can accept connections outside of localhost e.g. /etc/postgresql/9.4/main/postgresql.conf: `listen_addresses = '*'`
-3. Make sure your psql database accepts the Docker container e.g. /etc/postgresql/9.4/main/pg_hba.conf: `host ddw_metadata postgres 172.29.0.0/16 trust` -->
 
 ### Docker
 
-1. Create a persistent dev volume `docker volume create --name=metadata`
-2. Build your app `docker-compose up --build`
-3. Migrate the database. `docker-compose run web python manage.py migrate`
-4. Create a superuser. `docker-compose run web python manage.py createsuperuser`
-5. Install frontend dependencies `npm install`
-6. Bundle frontend code and collect static files `npm run build` or `npm run docker.dev` for local development
-7. Run the app. `docker-compose up`
+1. Make sure you're starting with a clean DB volume, so Docker knows to create the new User `docker-compose down` `docker volume rm metadata`
+2. Create a persistent dev volume `docker volume create --name=metadata`
+3. Build your app `docker-compose up --build`
```

Comment:
Should we change this one to `docker-compose build`. Current one makes the final step redundant since the app is already running.

Reply:
Will test this. But in the past I've found you need to do `up --build` to get the Django to build successfully.

Reply:
Negative. We still need `up --build` otherwise it immediately fails on migrate:

```
Is the server running on host "db" (172.20.0.2) and accepting
	TCP/IP connections on port 5432?
```